SQL/MySQL.sublime-syntax

```diff
@@ -0,0 +1,179 @@
+    - include: regexps
+
+  comments:
+    - meta_append: true
+    - match: '#'
+      scope: punctuation.definition.comment.sql
+      push: inside-number-sign-comment
+
+  double-dash-comments:
+    - meta_include_prototype: false
+    - match: '--(?=\s)'
+      scope: punctuation.definition.comment.sql
+      push: inside-double-dash-comment
+
+  strings:
+    - meta_append: true
+    - match: '"'
+      scope: punctuation.definition.string.begin.sql
+      push: inside-double-quoted-string
+    - include: begin-interpolation
+
+  inside-double-quoted-string:
+    - meta_include_prototype: false
+    - meta_scope: string.quoted.double.sql
+    - match: '""'
+      scope: constant.character.escape.sql
+    - match: '"'
+      scope: punctuation.definition.string.end.sql
+      pop: 1
+    - include: string-interpolation
+
+  begin-interpolation:
+    - match: '%\{'
+      scope: punctuation.definition.string.begin.sql
+      push: inside-interpolation
+
+  inside-interpolation:
+    - meta_include_prototype: false
+    - meta_scope: string.other.quoted.brackets.sql
+    - match: '\}'
+      scope: punctuation.definition.string.end.sql
+      pop: 1
+    - include: string-interpolation
+
+  string-interpolation:
+    - meta_include_prototype: false
+    - match: '(#\{)([^\}]*)(\})'
+      scope: string.interpolated.sql
+      captures:
+        1: punctuation.definition.string.begin.sql
+        3: punctuation.definition.string.end.sql
+
+  regexps:
+    - match: /(?=\S.*/)
+      scope: punctuation.definition.string.begin.sql
+      push: inside-regexp
+
+  inside-regexp:
+    - meta_include_prototype: false
+    - meta_scope: string.regexp.sql
+    - match: /
+      scope: punctuation.definition.string.end.sql
+      pop: 1
+    - include: string-interpolation
+    - match: \\/
+      scope: constant.character.escape.slash.sql
+
+  operators:
+    - meta_append: true
+    - match: \|\|
+      scope: keyword.operator.concatenation.sql
+
+  types:
+    - meta_append: true
+    - match: (?i:\bwith(?:out)?\s+time\s+zone\b)
+      scope: storage.type.sql
+
+  inside-number-sign-comment:
+    - meta_include_prototype: false
+    - meta_scope: comment.line.number-sign.sql
+    - match: \n
+      pop: 1
+
+  expressions:
+    - meta_prepend: true
+    - match: (?i)\b(?:CONCATENATE|CONVERT|LOWER|SUBSTRING|TRANSLATE|TRIM|UPPER)\b
+      scope: support.function.string.sql
+    - match: (?i)\b(?:using)\b
+      scope: keyword.other.mysql
+
+  statements:
+    - meta_append: true
+    - match: (?i:\b(begin(\s+work)?|start\s+transaction|commit(\s+work)?|rollback(\s+work)?)\b)
+      scope: keyword.other.LUW.sql
+
+  ddl-statements:
+    - meta_prepend: true
+    - match: \b(?i:create\s+or\s+replace)\b
+      scope: keyword.other.ddl.sql
+      push: [ddl-create-target, create-condition, ddl-target]
+
+  dml-statements:
+    - meta_append: true
+    - match: (?i:\binsert(\s+(?:ignore\s+)?into)?\b)
+      scope: keyword.other.DML.sql
```

Comment:
Why is `DML` capitalized ? Relic of the old syntax ?

Reply:
yup :)